### ISSUE TITLE
Forecast submissions through pull request while having an existing fork

### ISSUE THREAD

**Issue by scc-usc** [midas-network/covid19-scenario-modeling-hub#20]
Hi,

It seems like if the submitting account has already forked the parent project "covid19-forecast-hub", it cannot fork this one too. One can go around this by having the two forks on two separate accounts. Alternatively, each submitting team can clone this repo and then submit files. Not sure if we will need write-access for that. I wonder if you have any thoughts on this. I am not an expert in Git, so please let me know if I am missing something obvious. Being able to submit without cloning is slightly more convenient, but perhaps not supported as it would require creating a fork of a fork. If this repo were to be recreated without being a fork this issue can be avoided.

Thanks,
Ajitesh

**Comment by JohnLevander** [comment]
I would agree that recreating the repo, not as a fork, is the best way forward.  We’d lose the commit history, but that would be all I believe.  We really aren’t using it as a true fork anyway…I think we just wanted to copy the structure from the reichlab.   Let me know if I should do this...

**Comment by scc-usc** [comment]
Yes, a new repository seems to be the best approach to me too. There may be ways to preserve commit history, if needed: https://stackoverflow.com/questions/17371150/moving-git-repository-content-to-another-repository-preserving-history
